src/lib/objects.js

```diff
@@ -0,0 +1,75 @@
+const createPerson = (name, age) => {
+  const person = Object();
+  person.name = name;
+  person.age = age;
+  return person;
+};
+
+const getName = object => {
+  const name = object.name;
+  return name;
+};
+
+const getProperty = (property, object) => {
+  return object[property];
+};
+
+const hasProperty = (property, object) => {
+  return !!object[property];
+};
+
+const isOver65 = person => {
+  return person.age > 65;
+};
+
+const getAges = people => {
+  const ages = people.map(person => {
+    return person.age;
+  });
+  return ages;
+};
+
+const findByName = (name, people) => {
+  console.log(name);
+  const find = people.find(person => person.name === name);
+  console.log(find);
+  return find;
```

Comment:
You can return the result of the find function here:
```
  return people.find(person => person.name === name);
```
Also, you should always remove console logs when you are done with them. In this case these wont show up anywhere when running the tests as there no tests that cover any of the methods in this file, but the point still stands